Owncloud iOs Client/DataBase/Queries/ManageUploadsDB.m

```diff
@@ -272,6 +272,14 @@ +(void) setStatus:(NSInteger) status andKindOfError:(NSInteger) kindOfError byUp
     
     DLog(@"setStatus: %ld andKindOfError: %ld currentUpload: %@", (long)status, (long)kindOfError, currentUpload.uploadFileName);
     
+    if (status == 4) { //error uploading, refresh cell to remove uploading yellow arrow
```

Comment:
Not use use directly the numbers, use the errorUploading enum to know what is the comparation